### DISCUSSION TITLE
Could not connect to DFU interface

### DISCUSSION THREAD

**Discussion by sanjay-swain** [EdgeTX/buddy#68] [Q&A] [answered]
Hi, I was trying to upgrade the firmware but continuously face the following error. I tried installing drivers but nothing is working..
![image](https://github.com/EdgeTX/buddy/assets/59828852/4e52a407-86de-4ca0-a56b-4721d5970fb9)

**Comment by Muhammad-Hazimi-Yusri** [comment]
getting same issue here, using Jumper T-Pro V2

**Comment by Muhammad-Hazimi-Yusri** [comment]
after further checks, on about page, it shows my edgetx version to be 2.9.0-selfbuild (my first time getting a radio and just tinkering to update it according to guide, probably the seller flashed latest unstable version?), does this mean this issue is related to this latest commit? https://github.com/EdgeTX/edgetx/commit/f3a08b9e4f60f4f00af136b36915f804b0ae5bb3

I didn't realised for a while that 2.9.0 is bigger than 2.8.4 lol

**Comment by shermify** [comment, marked as answer]
Use zadig to install or reinstall winusb for the STM bootloader.

**Comment by njinco** [comment]
> Use zadig to install or reinstall winusb for the STM bootloader.

this worked for me. thanks for giving the link for the driver install 👍 I knew it was a driver problem couldn't find a legit source. Maybe this can be added somewhere in a wiki for future reference?
